contracts/MyMultiSig.cdc

```diff
@@ -272,19 +247,37 @@ pub contract MyMultiSigV2 {
             return self.signers
         }
 
-        pub fun getThreshold(): UInt64 {
+        pub fun getThreshold(): UInt {
             return self.threshold
         }
 
         pub fun getVerifiedSignersForAction(actionUUID: UInt64): {Address: Bool} {
             return self.borrowAction(actionUUID: actionUUID).accountsVerified
```

Comment:
i'm a bit confused with this naming now. 😅 
`accountsVerified` could be true/false/nil. `getVerifiedSignersForAction` makes me think this is to get all accounts that approved the action, but we are actually just getting all signers, right? shall we name it `getAllSignersForAction` ?

Same on FE, we are storing this data under https://github.com/DapperCollectives/VESSEL/blob/183db5f8733bbe69356a5518c0ff1f10cf7ab50c/packages/client/src/hooks/useTreasury.js#L293

Here instead of `verifiedSigners`, can we consider name it `allSigners`?

Reply:
@amyliu6 yes, I think thats a much better naming convention! i'll make that update.

Reply:
@amyliu6 I've updated the naming of `accountsVerified`/`verifiedSigners` to `signerResponses` and `getApprovedSignersForAction`/`getRejectedSignersForAction`.  In general I think we should move away from the term `verified` to describe signer status... it should be `approved`/`rejected`/`pending`.

i've also updated `signerResponses` type from `{Address: Bool}` to `{Address: SignerResponse}` where `SignerResponse` is an `enum` type.  The issue with using `Bool` is that it seems in Golang, if you try to access a `map[string]bool` where the key doesnt exist, you get `false` instead of `nil`, so it wasn't working as cleanly as I was hoping.  Additionally, this gives us more flexibility down the road to add more SignerResponse statuses (i.e. `abstain` is an option that similar products support).